### DISCUSSION TITLE
translate layout

### DISCUSSION THREAD

**Discussion by zakidze** [amannn/next-intl#236] [Q&A]
### Is your feature request related to a problem? Please describe.

if i have a layout  i cant use get static props bcause its component that wraps the page

### Describe the solution you'd like

i like to have specifique json file to tranlsate each layout or component on its own

### Describe alternatives you've considered

.

**Comment by amannn** [comment]
Hello @zakidze,

I'll convert this to a discussion since this is a usage question, not a feature request.

To briefly touch on the topic: You have to make sure that the provider wraps all the components that use the `useTranslations` hook, including your layout component.
